ui-testing-core/src/main/java/com/avito/android/test/espresso/action/recycler/RecyclerViewScroll.kt

```diff
@@ -185,8 +186,15 @@ private fun RecyclerView.scrollItemAtPositionToCenter(
     uiController: UiController,
     position: Int
 ) {
-    scrollToPosition(position)
+    try {
```

Comment:
Can we reuse this logic between lines 189 and 216? and ScrollToIfPossibleAction.kt

Reply:
No